src/sentry/models/options/organization_option.py

```diff
@@ -7,7 +7,7 @@
 from sentry.db.models import FlexibleForeignKey, Model, sane_repr
 from sentry.db.models.fields import EncryptedPickledObjectField
 from sentry.db.models.manager import OptionManager, Value
-from sentry.tasks.relay import schedule_update_config_cache
+from sentry.tasks.relay import schedule_invalidate_config_cache
```

Comment:
just to be clear, there are also instances in `getsentry` that have to be updated. but it's probably fine to deploy an intermediate state where those invalidations trigger the wrong task

Reply:
https://github.com/getsentry/getsentry/pull/7592